modules/python/test/test_misc.py

```diff
@@ -482,6 +482,17 @@ def test_reserved_keywords_are_transformed(self):
         )
 
 
+class CanUsePurePythonModuleFunction(NewOpenCVTests):
+    def test_can_get_ocv_version(self):
+        import sys
+        if sys.version_info[:2] < (3, 0):
+            raise unittest.SkipTest('Python 2.x is not supported')
+
+        self.assertEqual(cv.misc.get_ocv_version(), cv.__version__,
+                         "Can't get package version using Python misc module")
```

Comment:
It is still not clear to me that problem we want to resolve here (except loading from filesystem).

See here: https://github.com/alalek/opencv/commits/test_20611

Reply:
I don't quite understand this comment